hyperspy/__init__.py

```diff
@@ -39,7 +26,6 @@
 
 import Release
 
+__all__ = ["api"]
```

Comment:
Maybe, but this is actually an improvement over the current situation in which "from hyperspy import *" only imports the modules explicitly imported in `hyperspy/__init__.py` i.e. `Release`.

Actually, given that the API will be in `hyperspy.api` it doesn't make much sense to use "from hyperspy import *", so one wonders indeed if the current empty `__all__` was actually better. (Any opinions on this?)

Regarding hiding functionality, if someone wants to access the internal modules they can simply import them explicitly but the point is that there should be no need to import anything outside `hyperspy.api` for data analysis.